Makefile

```diff
@@ -557,6 +557,39 @@ doc-custom: install-doc release-version
+	kubectl delete -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/30bootstrap-service.yml
+	kubectl delete -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/50kafka.yml
+	kubectl delete -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-0.yml
+	kubectl delete -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-1.yml
+	kubectl delete -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-2.yml
+	
+launch-kafka:
+  kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/configure/docker-storageclass-broker.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/configure/docker-storageclass-zookeeper.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/00-namespace.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/zookeeper/10zookeeper-config.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/zookeeper/20pzoo-service.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/zookeeper/30service.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/zookeeper/50pzoo.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/zookeeper/51zoo.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/10broker-config.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/20dns.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/30bootstrap-service.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/kafka/50kafka.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-0.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-1.yml
+	kubectl apply -f https://raw.githubusercontent.com/Yolean/kubernetes-kafka/cd332860d3a8367de85e9b046b2a277536d3df4e/outside-services/outside-2.yml
```

Comment:
How would you feel about downloading these files into `etc/kafka` and then you could create / destroy them with a single line:

```
kubectl create -f etc/kafka -R
```

The `-R` flag will recursively walk the directory.

This has the advantage that it declutters our Makefile and our CI becomes a little more hermetic and won't start failing if `Yolean` decides to delete the repo.

If you wouldn't mind doing a little extra cleanup, I think it'd be nice to do the same for prometheus, those are already a single line but it'd be nice for them to be a bit more hermetic.